core/modules/views/tests/views_test/views_test.module

```diff
@@ -35,6 +35,7 @@ function views_test_autoload_info() {
     'views_test_plugin_access_test_dynamic' => 'test_plugins/views_test_plugin_access_test_dynamic.inc',
     'views_test_plugin_access_test_static' => 'test_plugins/views_test_plugin_access_test_static.inc',
     'views_test_plugin_style_test_mapping' => 'test_plugins/views_test_plugin_style_test_mapping.inc',
+
   );
```

Comment:
I guess, this removal of the empty line isn't intentional, but a leftover. If that gets reverted, the file won't show up here anymore.